lib/smart_answer/calculators/register_a_birth_calculator.rb

```diff
@@ -38,7 +38,7 @@ def paternity_declaration?
     end
 
     def before_july_2006?
-      Date.new(2006, 07, 01) > childs_date_of_birth
+      Date.new(2006, 0o7, 0o1) > childs_date_of_birth
```

Comment:
I guess this is technically right (which is why Rubocop makes this change automatically), but I'd suggest changing these numbers in to decimal instead.